server/cmd/mmctl/commands/user.go

```diff
@@ -142,7 +143,8 @@ var DeleteUsersCmd = &cobra.Command{
 	Use:   "delete [users]",
 	Short: "Delete users",
 	Long: `Permanently delete some users.
-Permanently deletes one or multiple users along with all related information including posts from the database.`,
+Permanently deletes one or multiple users along with all related information including posts from the database.
+In order to use this command ServiceSettings.EnableAPIUserDeletion must be set to true. See ` + constants.CONFIG_DOCUMENTATION_URL + ` for more information.`,
```

Comment:
I would prefer if the command checks the config and outputs an actionable error instead of having the static documentation in the help text.

Does that sound reasonable to you?

Reply:
ok!

Reply:
@hanzei , the problem I'm encountering is that I should load app with the right config in order to make a conditional output. How can I achieve this?